frontend/settingsTab.jsx

```diff
@@ -76,12 +81,12 @@ const SettingsStack = createStackNavigator();
 function SettingsTab() {
   return (
   // Implement a StackNavigator for the various settings
-    <SettingsStack.Navigator initialRouteName="Settings">
-      <SettingsStack.Screen name="Settings" component={Settings} />
-      <SettingsStack.Screen name="Preferences" component={Preferences} />
-      <SettingsStack.Screen name="Profile" component={Profile} />
-      <SettingsStack.Screen name="Notifications" component={Notifications} />
-      <SettingsStack.Screen name="Time Zone" component={TimeZone} />
+    <SettingsStack.Navigator>
+      <SettingsStack.Screen name="Settings" component={Settings} options={{ headerShown: false }} />
+      <SettingsStack.Screen name="Preferences" component={Preferences} options={{ headerTransparent: true }} />
+      <SettingsStack.Screen name="Profile" component={Profile} options={{ headerTransparent: true }} />
+      <SettingsStack.Screen name="Notifications" component={Notifications} options={{ headerTransparent: true }} />
+      <SettingsStack.Screen name="Time Zone" component={TimeZone} options={{ headerTransparent: true }} />
```

Comment:
Because the headerTransparent option is essentially the default options of the child Screens, this option should be raised up to the Navigator.

```suggestion
    <SettingsStack.Navigator screenOptions={{ headerTransparent: true }}>
      <SettingsStack.Screen name="Settings" component={Settings} options={{ headerShown: false }} />
      <SettingsStack.Screen name="Preferences" component={Preferences} />
      <SettingsStack.Screen name="Profile" component={Profile} />
      <SettingsStack.Screen name="Notifications" component={Notifications} />
      <SettingsStack.Screen name="Time Zone" component={TimeZone} />
```